### ISSUE TITLE
extension that sets all to false

### ISSUE THREAD

**Issue by monperrus** [amq/firefox-debloat#22]
Is there a Firefox  extension that sets all those properties to false?

**Comment by jomo** [comment]
Yes, there is [privacy settings](https://addons.mozilla.org/en-US/firefox/addon/privacy-settings/) (repo [here](https://github.com/schomery/privacy-settings/)).

**Comment by monperrus** [comment]
Excellent, thanks!

Should the two repos be merged for increased visibility?
